docs/UserGuide.md

```diff
@@ -1,44 +1,85 @@
 ---
 layout: page
 title: AcademyAssist User Guide
+
 ---
+# AcademyAssist User Guide
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+
+2. [Quick Start](#quick-start)
+
+3. [Features](#features)
+   * [Adding a student](#adding-a-student--add)
+   * [Deleting a student](#deleting-a-student--del)
+   * [Editing a student](#editing-a-student--edit)
+   * [Viewing all student](#viewing-all-students--view)
+   * [Finding a student](#finding-a-student--find)
+   * [Sorting students](#sorting-students--sort)
+   * [Clearing all entries](#clearing-all-entries--clear)
+   * [Viewing help](#viewing-help--help)
+   * [Exiting the program](#exiting-the-program--exit)
+
+4. [Data Management](#data-management-in-academyassist)
+
+5. [FAQ](#faq)
+
+6. [Command summary](#command-summary)
+
+## Introduction
+Welcome to AcademyAssist, your ultimate solution for efficient student contact management in tuition centers. 
+This desktop application is designed to streamline your administrative tasks, allowing you to focus more 
+on what truly matters - educating and nurturing young minds.
+
+AcademyAssist combines the power of a **Command Line Interface (CLI) with the user-friendly aspects of a Graphical 
+User Interface (GUI).** This hybrid approach ensures that you can manage your student database with 
+lightning-fast efficiency while still enjoying the visual benefits of modern software design. Whether you're a 
+seasoned administrator or new to digital management systems, AcademyAssist is intuitive enough for beginners yet 
+powerful enough for experts. With features like quick student addition, easy information retrieval, 
+and automated data saving, AcademyAssist is set to revolutionize how you manage your tuition center.
 
-AcademyAssist is a **desktop app for managing student contacts in a tuition center, optimized for use via a Command 
-Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, 
-AcademyAssist can get your contact management tasks done faster than traditional GUI apps.
+Let's embark on this journey to simpler, faster, and more effective student management!
+
+---
 
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
+   1. Open a terminal window, and navigate to the root of your project folder.
```

Comment:
Perhaps we can expand a bit on the explanation since a beginner may not know what is the root of the project folder